30-inheritance.cs

```diff
@@ -56,17 +55,16 @@ public string Calculate()
     {
         int average = testScores.Sum() / testScores.Length;
 
-        if (100 >= average && average >= 90)
+        if (average <= 100 && average >= 90)
             return "O";
-        if (90 > average && average >= 80)
+        if (average < 90 && average >= 80)
             return "E";
-        if (80 > average && average >= 70)
+        if (average < 80 && average >= 70)
             return "A";
-        if (70 > average && average >= 55)
+        if (average < 70 && average >= 55)
             return "P";
-        if (55 > average && average >= 40)
+        if (average < 55 && average >= 40)
             return "D";
-        else
-            return "T";
-    }
-}
+
+        return "T";
+    }
```

Comment:
In my opinion, this is more readable and more light, what do you think @hakhov ?
```
public string Calculate()
{
	int average = testScores.Sum() / testScores.Length;

	return average switch
	{
		<= 100 and >= 90 => "O",
		< 90 and >= 80 => "E",
		< 80 and >= 70 => "A",
		< 70 and >= 55 => "P",
		< 55 and >= 40 => "D",
		_ => "t",
	};
}
```